src/routes/(public)/(guest)/register/invite/[slug]/+page.svelte

```diff
@@ -77,7 +77,11 @@
                 await sdk.forConsole.account.updatePrefs(newPrefs);
                 await invalidate(Dependencies.ACCOUNT);
```

Comment:
unrelated but can we move this to `await Promise.all([...]);`, would be slightly quicker.